packages/website/ts/components/dialogs/ledger_config_dialog.tsx

```diff
@@ -199,7 +199,6 @@ export class LedgerConfigDialog extends React.Component<LedgerConfigDialogProps,
     }
     private _onAddressSelected(selectedRowIndexes: number[]) {
         const selectedRowIndex = selectedRowIndexes[0];
-        this.props.blockchain.updateLedgerDerivationIndex(selectedRowIndex);
```

Comment:
Did you manually test signing an order with a given Ledger address on website? Would be good to manually check that it's setting the correct address as the `from` parameter in the requests.

Reply:
I manually [tested allowance changes](https://kovan.etherscan.io/tx/0x8aac89484575241fe19f48c6644f5a55938b582675d4d11f3413a81263fda80b) in portal with the 4th address on the ledger which should cover `sendTransaction`. But I'll quickly test `signPersonalMessage` through signing an order just incase.